Presentations/06-R-programming-practices.Rmd

```diff
@@ -1020,7 +965,7 @@ print(plot)
 
 ## Why indent?
```

Comment:
This could go to the appendix, since it's more aesthetics that programming.

Reply:
I agree, thanks for the suggestion. Solved in 7d50441